frontend/src/pages/AdminVendorRequests.jsx

```
@@ -107,8 +109,81 @@ const VendorRequestPage = () => {
+            {/* Header */}
+            <div className="px-6 py-4 border-b bg-gradient-to-r from-blue-50 to-indigo-50">
+              <h3 className="text-lg font-semibold text-gray-800">Request Details</h3>
+            </div>
+            
+            {/* Content */}
+            <div className="p-6 bg-white">
+              <div className="space-y-6">
+                {/* Name row */}
+                <div className="grid grid-cols-2 gap-6">
+                  <div className="bg-gray-50 p-4 rounded-lg">
+                    <label className="block text-sm font-medium text-gray-600 mb-1">First Name</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)